src/utils/parser.cpp

```diff
@@ -107,4 +58,28 @@ std::vector<std::vector<std::string>> loadData(const std::string& path)
     return allRows;
 }
 
+void saveData(const std::string& path, const std::vector<std::vector<std::string>>& allRows)
+{
+    // Creating a ofstream object that opens outfile with the given path
+    std::ofstream outfile(path);
 
+    // Check if path file extension matches expected file extension
+    if(path.substr(path.size()-4) != ".csv"){throw BadFileFormat(QString::fromUtf8("Wrong File type\n"), QString::fromStdString(path));}
+    // If file doesn't open throw BadFile error
+    if (!outfile.is_open()){throw BadFile(QFile(QString::fromStdString(path)));}
+    for(std::vector<std::string> columns: allRows)
+    {
+        /* Output first column before entering loop, first column doesn't need a comma preceding it, erase first column */
+        outfile << columns.front();
```

Comment:
Can we get a check for checking if the string contains a comma? When the string contains a comma, let's output quotations marks first and faster.

Reply:
None of them have commas, I'm putting the comma as I output to the file. None of the first items in the row need a comma before outputting

Reply:
I'm talking about if the string contains a comma: `Arkansas, Ohio`. This needs to be wrapped in quotation marks: `"Arkansas, Ohio"`.

Reply:
They are, everything has been tested, matches original csv

Reply:
But that is because you hard-code the `""` in your `saveToFile()` for location. You won't always know if you have a comma in your string unless you check.

Reply:
Chris, why, let's push this through, this pr should have gone though already, it's Friday and I should be working on the next thing

Reply:
Because it's going to cause problems later on. If an admin changes a souvenir/stadium name to `The best, buy me`, our parser is going to have a heart attack.